docs/blog/new-community-coc-2019.rst

```diff
@@ -0,0 +1,22 @@
+.. post:: Feb 1, 2019
+   :tags: community, code of conduct, announcements
+
+Improved Community Code of Conduct
+==================================
+
+As you may know, we aim to make Write the Docs an incredibly diverse, friendly, and inclusive community. As part of that effort, we have a community :doc:`/code-of-conduct`. We have refactored and improved our Code of Conduct, based on lessons in our own and other communities, and how our community has developed in recent years.
+
+The most significant changes we've made are:
+
+* The CoC now explains the fundamental goals we have with a CoC.
```

Comment:
I'd prefer all of those points to be more like "we've clarified the fundamental CoC goal. We've added clarification.... 

What do you think?

Reply:
Yeah, one way around that is to split the list:

We've added the following
* A
* B

And we've updated:
*C
*D